### ISSUE TITLE
Cannot start chromium

### ISSUE THREAD

**Issue by conorsch** [subgraph/subgraph-os-issues#203]
Running the latest version of `chromium`, installed from the Debian Stretch repos, is essentially a no-op. Using a fresh install of the latest SubgraphOS iso, `subgraph-os-alpha_2016-12-30_1.iso`. Chromium installs fine, but the oz sandbox fails to run. 

Version info:

```
[user@subgraph:~] $ apt-cache show chromium | head -n 6
Package: chromium
Source: chromium-browser
Version: 55.0.2883.75-3
Installed-Size: 160895
Maintainer: Debian Chromium Maintainers <pkg-chromium-maint@lists.alioth.debian.org>
Architecture: amd64
```

The oz logs, taken from `oz log | grep chromium`, are a bit more verbose, so [see this gist](https://gist.github.com/conorsch/8e9c39c953f6dcd68488021111ae137e). Comparing the oz logs to dmesg output, the only notable difference is the presence of a grsec core dump prohibition in dmesg but I don't expect that's causing the failure. Still, here it is:

```
[53555.542239] grsec: denied resource overstep by requesting 4096 for RLIMIT_CORE against limit 0 for /srv/oz/rootfs/usr/lib/chromium/chromium[chromium:11110] uid/euid:1000/1000 gid/egid:1000/1000, parent /usr/bin/oz-init[oz-init:11056] uid/euid:0/0 gid/egid:0/0
```

I've not made any customizations to `/etc/security/limits.conf`.

**Comment by dma** [comment]
Yup, the problem is actually Chromium setuid sandbox executable. We modified Oz to make an exception for it. You'll need a new oz-daemon deb. We'll try to make one very soon you can try.

**Comment by conorsch** [comment]
Roger that, happy to test a build!

**Comment by korolevskaya** [comment]
It is interesting is this solution secure as browsers actually vulnerable in terms of reasons of sandboxing if i understand.

**Comment by conorsch** [comment]
Still hitting this problem. Would gladly test rc packages and report results.

**Comment by dma** [comment]
I'll get you a package today or tomorrow to test. Do you use any OpenVPN providers? You can try that too.

**Comment by dma** [comment]
Also the clearnet bridge for dealing with captive portals. We'll be using Chromium for that.

**Comment by conorsch** [comment]
>Do you use any OpenVPN providers?

Totally! Looking forward to testing that as well.
